src/api/route-services/pipeline-response.js

```diff
@@ -0,0 +1,41 @@
+const AWS = require('../../utils/requireAWS');
+const validateRequest = require('../../utils/schema-validator');
+
+const pipelineResponse = async (io, message) => {
+  await validateRequest(message, 'PipelineResponse.v1.yaml');
+
+  // Fail hard if there was an error.
+  const { response: { error } } = message;
+
+  if (error) {
+    throw new Error(error);
+  }
+
+  // Download output from S3.
+  const s3 = new AWS.S3();
```

Comment:
don't we already use the s3 client in the API for downloading too big files from SNS? Because if we have the implementation of downloading of files in different places, it will be worth to have the implementation in one place and just reused

Reply:
We sort of use a similar pattern, but that one is more involved as it involves looking through all the result objects and downloading only those which are required to be downloaded. This one is much simpler so there was not a lot of things to factor in together.

Reply:
makes senes in that case